packages/database/src/dbDotEnv.js

```diff
@@ -1,27 +1,55 @@
+import { readFileSync, existsSync } from "node:fs";
```

Comment:
@maparent `turbo dev` is now failing at this line

<img width="1434" height="1669" alt="image" src="https://github.com/user-attachments/assets/3689516a-2936-438d-bfcd-5a176d7b021e" />

Reply:
Ouch.  Works for me, and in deployment so really puzzled. Let's review your setup first thing tomorrow.